code/game/objects/effects/particles_system.dm

```diff
@@ -111,10 +131,48 @@
 	icon_state = "blank"
 	var/atom/main_holder
 
+/obj/abstract/particles_holder/proc/special_setup(var/particle_string)
+	switch(particle_string)
+		if ("Candle")
+			appearance_flags = RESET_COLOR
+			blend_mode = BLEND_ADD
+			plane = ABOVE_LIGHTING_PLANE
+		if ("Candle2")
+			appearance_flags = RESET_COLOR
+			blend_mode = BLEND_ADD
+			plane = ABOVE_LIGHTING_PLANE
+		if ("Cult Gauge")
+			plane = HUD_PLANE
+			layer = MIND_UI_BUTTON+0.5
+		if ("Cult Smoke")
+			plane = FLOAT_PLANE
+		if ("Cult Smoke2")
+			plane = FLOAT_PLANE
+		if ("Cult Halo")
+			plane = ABOVE_LIGHTING_PLANE
+		if ("Nar-SieHasRisen1")
+			plane = ABOVE_HUD_PLANE
+		if ("Nar-SieHasRisen2")
+			plane = ABOVE_HUD_PLANE
+		if ("Nar-SieHasRisen3")
+			plane = ABOVE_HUD_PLANE
+
 //////////////////////////////////////PARTICLES///////////////////////////////////
 
 var/list/particle_string_to_type = list(
 	"Steam" = /particles/steam,
+	"Tear Reality" = /particles/tear_reality,
```

Comment:
well I can think of 3 reasons:
* particles need a string key anyway to be listed in the object's particle_systems list. If we used the type there instead we'd also have to type the type each time we call adjust_particles() and the likes, which brings to the second point:
* code readability, frankly I'd rather look at this
![image](https://github.com/vgstation-coders/vgstation13/assets/7573912/5b5cde16-f548-4a75-a00a-e5ca807884c3)
than at this
![image](https://github.com/vgstation-coders/vgstation13/assets/7573912/eb21d291-a7db-4700-954b-1606476b0b89)
* third reason is that it it's easier to varedit and type for admins, rather than having to dig through type lists.

Reply:
but alright fine